dora/core/common/src/main/java/alluxio/conf/RemovedKey.java

```diff
@@ -140,8 +140,6 @@ private static String replacedSince(String version, String newProperty) {
           PropertyKey.Name.WORKER_RAMDISK_SIZE));
       put("alluxio.worker.network.netty.async.cache.manager.threads.max", removedSince(V2_0_0));
       put("alluxio.worker.network.netty.backlog", removedSince(V2_0_0));
-      put("alluxio.worker.network.netty.block.reader.threads.max", removedSince(V2_0_0));
-      put("alluxio.worker.network.netty.block.writer.threads.max", removedSince(V2_0_0));
```

Comment:
this one is interesting...
These keys were removed in 2.0. Then in 3.0 we brought them back. Then we renamed them in this PR...